examples/dialogs/dialogs/app.py

```diff
@@ -50,13 +50,23 @@ def action_open_file_dialog_multi(self, widget):
 
     def action_select_folder_dialog(self, widget):
         try:
-            path_name = self.main_window.select_folder_dialog(
-                title="Select folder with Toga",
+            path_names = self.main_window.select_folder_dialog(
+                title="Select folder with Toga"
             )
-            self.label.text = "Folder selected:" + path_name
+            self.label.text = "Folder selected:" + ','.join([path for path in path_names])
         except ValueError:
             self.label.text = "Folder select dialog was canceled"
 
+    def action_select_folder_dialog_multi(self, widget):
+        try:
+            path_names = self.main_window.select_folder_dialog(
+                title="Select multiple folders with Toga",
```

Comment:
This is user-confusing, I'd bet. The user of a Toga app has no idea whether or not that app was made with Toga, they only know about the app name.

You can try to pull in the app name, or make this only "Select folders"